schemas/common_definitions.json

```diff
@@ -209,6 +214,46 @@
     "additionalProperties": false,
     "required": ["years"]
   },
+  "value_reference": {
```

Comment:
Is this needed?. Can we not reuse https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/schemas/string_interpolation/definitions.json#L137-L169?

Reply:
That reference also allows collection_metadata and list as potential sources, which aren't necessary at the moment for answer validation